.changeset/good-walls-film.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+Fix regression in sync-store getLogEvents `lastCheckpoint` that was introduced after splitting out the query into 2.
```

Comment:
You and I know what this means, but it's IMO a bit too low-level for the public changelog. It's a hard changeset to write because we're not certain that it caused any bugs. How about:
```
Fixed a regression that may have caused bugs related to indexing progress or event ordering.
```